content/coop.html

```diff
@@ -405,10 +405,10 @@ <h3> Side projects </h3>
 <h3> Admission average </h3>
 <p class="half">
   <span class="big">
-  Respondents who had an admission average above 95% earned 3.7% more on their 1st coop and 23.8% more overall than those that
+  Respondents who had an admission average above 95% earned 3.7% more on their 1st co-op and 23.8% more overall than those that
   had a lower entrance average.
   </span>
-  Based on the coop average in first year, it is unlikely that admission are a causation of average salary.
+  Based on the co-op average in first year, it is unlikely that admission are a causation of average co-op salary.
```

Comment:
```suggestion
  Based on the co-op average in first year, it is unlikely that admission averages are a causation of average co-op salary.
```

or we're missing a word; judging by context, i'm guessing it's "admission grades/averages" ?